File: python/docs_code_snippets/spans-create.py
from datetime import datetime, timezone
from opperai import Opper

opper = Opper()

# --- docs ---
span = opper.spans.create(
    name="my-pipeline",
    start_time=datetime.now(timezone.utc).isoformat(),
    input="Starting the pipeline",
    meta={"userId": "u-123"},
)
print(f"Span: {span.id}, Trace: {span.trace_id}")
# --- /docs ---

# Cleanup
opper.spans.delete(span.id)

File: typescript/docs_code_snippets/spans-create.ts
import { Opper } from "opperai";

const opper = new Opper();

const span = await opper.spans.create({
  name: "my-pipeline",
  start_time: new Date().toISOString(),
  input: "Starting the pipeline",
  meta: { userId: "u-123" },
});
console.log(`Span: ${span.id}, Trace: ${span.trace_id}`);
